extensions/control-plane/api/management-api/edr-cache-api/src/main/resources/META-INF/services/org.eclipse.edc.spi.system.ServiceExtension

```diff
@@ -0,0 +1,14 @@
+#
+#  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Contributors:
+#       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+#
+#
+
```

Comment:
the extension class reference is missing

Reply:
Thanks @ndr-brt  good catch :)